### DISCUSSION TITLE
Accent circonflexe ne fonctionne pas - ê

### DISCUSSION THREAD

**Discussion by madiwas** [keepassxreboot/keepassxc#6617] [Q&A] [answered]
Bonjour,
J'utilise Keepass 2.6.2 sur Debian 11 64 bits, et je rencontre un bug sur <ê> il n'est pas reconnu dans l'ouverture de ma base de données kdbx.
Si j'affiche le mot de passe tapé la lettre ê n'est s'affiche e, je la corrige, et là keepassxc l'intègre.
Avez-vous déjà rencontré ce phénomène ?
Bien cordialement,
Madiwas

**Comment by droidmonkey** [comment, marked as answer]
C’est un bug avec Qt et nous n’avons aucun contrôle dessus malheureusement
